### ISSUE TITLE
Run a test build of the platform api/javadoc using Java 11 to see what issues might exist.

### ISSUE THREAD

**Issue by starksm64** [eclipse-ee4j/jakartaee-api#110]
**Is your feature request related to a problem? Please describe.**
We know there are problems with moving from Java 8 to Java 11 javadoc generation, so we need to know what the scope of the problem is.

**Describe the solution you'd like**
Many spec project that have updated to Java 11 have seen errors. They either had to disable certain features or fix the javadoc source.

**Describe alternatives you've considered**
None.

**Additional context**
It was noted on the Aug 31 2021 platform call that we should start looking at this to see what the scope of the problem might be.

**Comment by kwsutter** [comment]
I'm going to transfer this Issue to the jakartaee-api repository since that's where we track the Platform API and Javadoc from.

**Comment by kwsutter** [comment]
This Issue is related to Issue #107.  Some of the work was completed via PR #109.  The Jenkins build was already updated to build with Java 11.  More scrubbing of the warnings produced is still probably warranted.  But, I think it's in pretty good shape.

**Comment by kwsutter** [comment]
Triaging the warnings that are generated from the javadoc generation.  Here are the Issues that I have created or added a comment to...

- https://github.com/eclipse-ee4j/security-api/issues/149 (referencing jacc (platform) in the webprofile javadoc)
- remove `<tag>` for implSpec in our pom.xml
- https://github.com/eclipse-ee4j/mail/issues/565 (referencing links outside of jakarta proper)

**Comment by kwsutter** [comment]
There are also these warnings, but these should be expected since we are not defining modules at the uber jar level for Jakarta EE 10:

```
[INFO] --- maven-javadoc-plugin:3.1.0:javadoc (default-cli) @ jakarta.jakartaee-web-api ---
[WARNING] Unable to derive module descriptor for /Users/sutter@us.ibm.com/git/jakartaee-api/jakartaee-web-api/target/jakarta.jakartaee-web-api-10-SNAPSHOT.jar
```
```
[INFO] --- maven-javadoc-plugin:3.1.0:javadoc (default-cli) @ jakarta.jakartaee-api ---
[WARNING] Unable to derive module descriptor for /Users/sutter@us.ibm.com/git/jakartaee-api/jakartaee-api/target/jakarta.jakartaee-api-10-SNAPSHOT.jar
```

**Comment by lukasj** [comment]
mail is supposed to be fixed already in 2.1.0-RC1 (available in central)
